source/comparison.rst

```diff
@@ -27,35 +20,40 @@ The following table will help you evaluate the differences.
        * :ref:`inmemory`
      - * WiredTiger_ (default)
        * In-Memory_ (Enterprise only)
+   * - Encryption-at-Rest
+     - * Key server = Hashicorp Vault
+       * Fully opensource
+     - * Key server = KMIP
+       * (Enterprise only)
    * - :ref:`Hot Backup <hot-backup>`
-     - YES for WiredTiger_
-     - NO
-   * - :ref:`Audit Logging <audit-log>`
+     - YES (replicaset)
+     - NO
+   * - LDAP Authentication
+     - * Simple LDAP Auth
+       * (legacy) :ref:`External SASL Authentication <ext-auth>`
+     - * Enterprise only
+       * Enterprise only
+   * - LDAP Authorization
      - YES
      - Enterprise only
-   * - :ref:`External SASL Authentication <ext-auth>`
+   * - Kerberos Authentication
      - YES
      - Enterprise only
-   * - :ref:`rate-limit`
-     - YES
-     - YES [#]_
-   * - Geospatial Indexes
-     - YES
-     - YES
-   * - Text Search
+   * - :ref:`Audit Logging <audit-log>`
      - YES
+     - Enterprise only
+   * - Log redaction
      - YES
-   * - ACID and MVCC compliant
-     - NO
-     - NO
-   * - Clustering Key Support
-     - NO
-     - NO
-   * - Sharding with Clustering Keys
-     - NO
-     - NO
-   * - Point-in-time Recovery
-     - NO
      - Enterprise only
+   * - SNMP Monitoring
+     - No
+     - Enterprise only
+
+Profiling Rate Limiting
+-----------------------
+
+Profiling Rate Limiting was added to PSMDB in v3.4 with --rateLimit argument. Since v3.6 MongoDB community (and Enterprise) reproduced this feature but called the option slowOpSampleRate_. Percona Server for MongoDB v3.6+ includes both. The two options have the same effect upon profiling and the database diagnostic log but the argument value is inverted: slowOpSampleRate = (1/rateLimit).
```

Comment:
```suggestion
Profiling Rate Limiting was added to |PSMDB| in v3.4 with ``--rateLimit`` argument. Since v3.6 MongoDB Community (and Enterprise) Edition reproduced this feature but called the option slowOpSampleRate_. |PSMDB| v3.6+ includes both. The two options have the same effect upon profiling and the database diagnostic log but the argument value is inverted: ``slowOpSampleRate`` = (1/``rateLimit``).
```